src/custom/components/Header/URLWarning/index.tsx

```diff
@@ -1,9 +1,42 @@
-import { PRODUCTION_URL } from 'constants/index'
 import React from 'react'
-import URLWarningUni from './URLWarningMod'
+
+import { PRODUCTION_URL } from 'constants/index'
+import { AlertTriangle } from 'react-feather'
+import URLWarningUni, { StyledClose } from './URLWarningMod'
+import ReactMarkdown from 'react-markdown'
+import { useAnnouncementVisible, useCloseAnnouncement } from 'state/profile/hooks'
+import { hashCode } from 'utils/misc'
+import useFetchFile from 'hooks/useFetchFile'
 
 export * from './URLWarningMod'
 
+// Announcement content: Modify this file to edit the announcement
+//  https://github.com/gnosis/cowswap/blob/announcements/docs/announcements.md
+const ANNOUNCEMENTS_MARKDOWN_URL =
```

Comment:
should this be in `consts` file no?

Reply:
imo can stay for now there. It's closer to the implementation and it's only used there, so kind of scoped. 
If for example we evolve it into using environment based logic, or other more complex logics, we would need to complicate the constants file for little benefit (no other place of the app needs this, so it can be considered implementation details).

Not strong opinion though, u can convince me if u insist a bit more :)